build.gradle

```diff
@@ -2,7 +2,7 @@ import net.ltgt.gradle.errorprone.CheckSeverity
 
 buildscript {
   ext.versions = [
-      'airline': '0.8',
+      'airline': '2.6.0',
```

Comment:
I had issues with Airline previously with mutable lists in Kotlin, so followed a known working approach.